oauth2_http/java/com/google/auth/oauth2/CloudShellCredentials.java

```diff
@@ -67,8 +67,9 @@ public static CloudShellCredentials create(int authPort) {
     return CloudShellCredentials.newBuilder().setAuthPort(authPort).build();
   }
 
-  private CloudShellCredentials(int authPort) {
-    this.authPort = authPort;
+  private CloudShellCredentials(Builder builder) {
+    super(builder);
+    this.authPort = builder.getAuthPort();
```

Comment:
Could you please test this as well? While it is partially covered in CredentialsProvider - we want to have coverage here as well. CredentialsProvider might someday go away or instantiate credentials differently.